File: beginner/__1000__/1020/1020.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) throws IOException {

        Scanner input = new Scanner(System.in);

        int num = input.nextInt();

        int age = num / 365;
        int rest = (num - (365 * age));

        int month = (rest / 30);
        int days = (rest - (30 * month));

        System.out.println(age + " ano(s)");
        System.out.println(month + " mes(es)");
        System.out.println(days + " dia(s)");
    }
}

File: beginner/__1000__/1020/1020.ts
const fs = require('fs')
const input = fs.readFileSync('/dev/stdin', 'utf-8')
const num = Number(input)

function ageTodays(n) {

    let age = Math.floor(n / 365);
    let rest = (n - (365 * age));

    let month = Math.floor(rest / 30);
    let days = (rest - (30 * month));

    console.log(`${age} ano(s)`);
    console.log(`${month} mes(es)`);
    console.log(`${days} dia(s)`);

}

ageTodays(num);